fs/vfs/fs_uio.c

```diff
@@ -84,6 +84,7 @@ void uio_advance(FAR struct uio *uio, size_t sz)
   int iovcnt = uio->uio_iovcnt;
   size_t offset_in_iov = uio->uio_offset_in_iov;
 
+  DEBUGASSERT(sz <= uio_resid(uio));
```

Comment:
merge tl the patch which add uio_advance

Reply:
done